changelog/dmd.verror_context.dd

```diff
@@ -0,0 +1,33 @@
+Make `verror=context` Usage Global and Update Related Test Files
+
+The `verror=context` error handling mechanism has been made global, ensuring that error messages now consistently include context information across the entire codebase. Previously, error handling was localized, leading to some errors lacking context in the output.
```

Comment:
This is a weird description: all it does is print the line of code the error applies to under the message. It has nothing to do with 'across the entire codebase' vs 'localized'.